### ISSUE TITLE
Creating packages per user even global packages disabled.

### ISSUE THREAD

**Issue by lunny** [go-gitea/gitea#24372]
### Feature Description

It would be more flexible.

**Comment by KN4CK3R** [comment]
You mean allowing single users/orgs to publish packages even if the feature is disabled?

**Comment by lunny** [comment]
> You mean allowing single users/orgs to publish packages even if the feature is disabled?

Yes. Or we have a default disabled and admin could change them in admin panel per user.
